reports/m1_proposal.md

```diff
@@ -0,0 +1,21 @@
+**Description of the data**
+
+The original raw data is downloaded from this URL <https://thepulseofthenation.com/#future>. This dataset is the pull result from Card Against Humanity to the nationally representative of American public about popular social concerns during time period August 1st to 15th 2018. The survey was conducted via face to face, cell phone, and land lines, and respondents' demographic info and opinion choices on survey questions were collected.
```

Comment:
should be poll not pull

Reply:
-let's mention the data is for August 2018 here instead of at the end? 
-Also maybe let's touch on the fact that we considered aggregating the monthly data available on the website but it was not possible since the polls change on a monthly basis, hence the latest poll data was used.

Reply:
Thanks for pointing out. Will update the content